src/components/20-molecules/file-upload/index.js

```diff
@@ -34,6 +34,14 @@ const IMAGE_FILE_TYPES = 'image/jpg, image/jpeg, image/png';
 
 export const getBytesFromKilobyte = kilobyte => 1024 * kilobyte;
 
+const findFileById = (files, file, indexOnly) => {
+  for (let i = 0, n = files.length; i < n; i++) {
+    if (files[i].id === file.id) {
+      return indexOnly ? i : files[i];
+    }
+  }
+  return undefined;
```

Comment:
This is the default return value, so why do you make it explicit?

Reply:
For clarity (I am going to test against undefined in line 553) and symmetry wit the other _explicit_ `return`.